stock_request_picking_type/models/stock_picking_type.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Open Source Integrators
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    code = fields.Selection(
+        selection_add=[("stock_request_order", "Stock Request Order")]
+    )
+    count_sr_todo = fields.Integer(string="To Do", compute="_compute_sr_count")
+    count_sr_open = fields.Integer(string="In Progress", compute="_compute_sr_count")
+    count_sr_late = fields.Integer(string="Late", compute="_compute_sr_count")
+
+    def _compute_sr_count(self):
```

Comment:
Compute method needs explicit values for all records

Reply:
@RLeeOSI Thank you for your review. Edited.